settings.gradle.kts

```diff
@@ -6,6 +6,7 @@ pluginManagement {
         google()
         gradlePluginPortal()
         mavenCentral()
+        maven("https://s01.oss.sonatype.org/content/repositories/snapshots") //KOTEST snapshot
```

Comment:
I'm not a fan of allowing snapshot repositories for releases, but if this can't be circumvented then so be it

Reply:
nope. not needed, but iOS tests are failing